packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsCore.tsx

```diff
@@ -146,17 +147,24 @@ export class SuggestionsCore<T> extends BaseComponent<ISuggestionsCoreProps<T>,
             role="listitem"
             aria-label={suggestion.ariaLabel}
           >
-            <TypedSuggestionsItem
-              id={'sug-item' + index}
-              suggestionModel={suggestion}
-              // tslint:disable-next-line:no-any
-              RenderSuggestion={onRenderSuggestion as any}
-              onClick={this._onClickTypedSuggestionsItem(suggestion.item, index)}
-              className={suggestionsItemClassName}
-              showRemoveButton={showRemoveButtons}
-              onRemoveItem={this._onRemoveTypedSuggestionsItem(suggestion.item, index)}
-              isSelectedOverride={index === this.currentIndex}
-            />
+            {renderSuggestionsItemAsDiv && onRenderSuggestion ? (
+              onRenderSuggestion(suggestion.item, {
```

Comment:
This doesn't seem to take into account the suggestionItemClassName, does that need to be included?